timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -175,19 +174,19 @@ def run(self):
         return output_string
 
 
-class LinuxRulesSigmaPlugin(SigmaPlugin):
+class SampleRulesSigmaPlugin(SigmaPlugin):
     """Sigma plugin to run Linux rules."""
 
-    _RULES_PATH = '../../../data/sigma/rules/linux'
+    _RULES_PATH = '../../../data/sigma_sample'
 
-    NAME = 'sigma_linux'
+    NAME = 'sigma_sample'
 
-class WindowsRulesSigmaPlugin(SigmaPlugin):
+class AllRulesSigmaPlugin(SigmaPlugin):
     """Sigma plugin to run Windows rules."""
 
-    _RULES_PATH = '../../../data/sigma/rules/windows'
+    _RULES_PATH = '../../../data/sigma/rules/'
 
-    NAME = 'sigma_windows'
+    NAME = 'sigma_all'
 
 class TestRulesSigmaPlugin(SigmaPlugin):
```

Comment:
Same here - do we want this to be visible in the UI?

Reply:
resolved by removing